tests/unit/10_certs.bats

```diff
@@ -33,6 +33,13 @@ teardown() {
   assert_success
 }
 
+@test "(certs) certs:add with multiple dots in the filename" {
+  run bash -c "dokku certs:add $TEST_APP $BATS_TMPDIR/tls/domain.com.crt $BATS_TMPDIR/tls/domain.com.key"
```

Comment:
Where does files like server.crt and server.key are in this repository, GitHub search sucks :\